### DISCUSSION TITLE
Can't keep zoom at high level

### DISCUSSION THREAD

**Discussion by nmccullough3** [osmandapp/OsmAnd#13062] [Q&A]
Apologies if I am not describing this very well. 

I am running OsmAnd maps on iOS and want to have the map move with me while zoomed out at a mile or two. I want to be able to have a wide overview of where a track goes and see the neighbouring villages etc. 

My problem is that when I zoom out after a few seconds the screen zooms in again. 

What am I doing wrong?

**Comment by dmpr0** [comment]
Hi, @nmccullough3!
If you use navigation, you can check "Map during navigation" settings, in Menu – Settings – Profiles – Navigation settings – Map during navigation. You need to change the **Auto-zoom** map to **No auto-zoom**.

**Reply by nmccullough3** [reply]
Ah - thank you so much. I thought I had searched everywhere but obviously not! Much appreciated.
